test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua

```diff
@@ -0,0 +1,57 @@
+local t = require('luatest')
+local cluster = require('luatest.replica_set')
+local server = require('luatest.server')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.cluster = cluster:new()
+    cg.master = cg.cluster:build_server({alias = 'master'})
+    cg.replica = cg.cluster:build_server(
+            {
+                alias = 'replica',
+                box_cfg = {
+                    replication = server.build_listen_uri('master'),
+                    memtx_use_mvcc_engine = true,
+                    txn_isolation = 'read-confirmed',
+                }
+            })
+    cg.cluster:add_server(cg.master)
```

Comment:
BTW, you can use `build_and_add_server` instead of `build_server`, `add_server`.

Reply:
Thanks for the suggestion, applied!